internal/ui/restore/json.go

```diff
@@ -108,6 +110,7 @@ type statusUpdate struct {
 	TotalFiles     uint64  `json:"total_files,omitempty"`
 	FilesRestored  uint64  `json:"files_restored,omitempty"`
 	FilesSkipped   uint64  `json:"files_skipped,omitempty"`
+	FilesDeleted   uint64  `json:"files_deleted,omitempty"`
```

Comment:
The new field must be documented at `doc/075_scripting.rst`.

Reply:
I have updated the documentation with the `files_deleted` json field.